packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -232,6 +238,7 @@ const GridFilterForm = React.forwardRef<HTMLDivElement, GridFilterFormProps>(
     const classes = useUtilityClasses(rootProps);
     const valueRef = React.useRef<any>(null);
     const filterSelectorRef = React.useRef<HTMLInputElement>(null);
+    const multiFilterOperator = filterModel.logicOperator ?? GridLogicOperator.And;
```

Comment:
Solves https://codesandbox.io/p/sandbox/zealous-shamir-jktxkz?file=%2Fsrc%2FDemo.tsx%3A20%2C11 i.e `filterModel.logicOperator` is `undefined` on initializing multiple filters. This issue is on next and not on master. Let me know if there's a better fix you are aware of.

Reply:
Shouldn't it be `Or` by default? https://github.com/mui/mui-x/blob/243ae70b89855bcaf58d665c4818758e1bea09a3/packages/grid/x-data-grid/src/models/gridFilterModel.ts#L16-L18

Reply:
Interesting point. @cherniavskii 
I feel this `@default GridLogicOperator.Or` statement is incorrect. Because if you check the current behavior on https://mui.com/x/react-data-grid/filtering/multi-filters/ it's `GridLogicOperator.And` (try adding 2-3 filters without touching logic operator)
Any suggestions if I should update the default value on this prop or the default behavior on the attached demo is incorrect?

Reply:
Indeed, here we set the default logic operator:

https://github.com/mui/mui-x/blob/69f1ca43a31f0dec0f0702a6959c7b3106dc9941/packages/grid/x-data-grid/src/hooks/features/filter/gridFilterState.ts#L8-L10

Feel free to update the jsdoc to reflect the actual default value 👍🏻

Reply:
@WillCDev Which prop are you referring to?